api/apps/api/src/modules/projects/blm/change-blm-range.handler.ts

```diff
@@ -1,18 +1,82 @@
 import { CommandHandler, IInferredCommandHandler } from '@nestjs/cqrs';
-import { Either, right } from 'fp-ts/Either';
+import { Either, isLeft, left, right } from 'fp-ts/Either';
 
-import { ChangeBlmRange, ChangeRangeErrors } from './change-blm-range.command';
+import {
+  ChangeBlmRange,
+  ChangeRangeErrors,
+  queryFailure,
+} from './change-blm-range.command';
 import { ProjectBlmRepo } from '@marxan-api/modules/blm';
+import { InjectEntityManager, InjectRepository } from '@nestjs/typeorm';
+import { Project } from '@marxan-api/modules/projects/project.api.entity';
+import { EntityManager, Repository } from 'typeorm';
+import { DbConnections } from '@marxan-api/ormconfig.connections';
+import { Logger } from '@nestjs/common';
+import { SetProjectBlm } from '@marxan-api/modules/projects/blm/set-project-blm';
+import { BlmValuesCalculator } from '@marxan-api/modules/projects/blm/domain/blm-values-calculator';
```

Comment:
Please pay attention to imports. As we are using barrel files, it may lead to unintended issues during compilation - also, it is nice to ensure what we import and from where ;)

Reply:
Yep, fixed!